libs/openchallenges/ui/src/lib/navbar/navbar.component.html

```diff
@@ -14,6 +14,7 @@
 
   <div class="flex-spacer"></div>
 
+  <a mat-button class="navbar-item" routerLink="/team">Team</a>
```

Comment:
Keep it consistent with the footer? "Meet Our Team"

Reply:
I added a previous for "Meet Our Team". Do you prefer this option?

Reply:
Looks good! Thanks